packages/app-runtime/test/lib/TestUtil.ts

```diff
@@ -300,19 +300,21 @@ export class TestUtil {
      * specified in the `until` callback is met.
      */
     public static async syncUntil(session: LocalAccountSession, until: (syncResult: SyncEverythingResponse) => boolean): Promise<SyncEverythingResponse> {
-        const syncResult = await session.transportServices.account.syncEverything();
-
-        const { messages, relationships } = syncResult.value;
-        const syncResponse = { relationships: [...relationships], messages: [...messages] };
+        const syncResponse: any = {};
```

Comment:
I made the method generic so that in future it dose not need to be changed when we add new properties to `SyncEverythingResponse`. The any is used because otherwise there will be TS errors.

Reply:
@sebbi08 this still seems to be unresolved.

Reply:
yea missed the app runtime when reverting the changes